rotkehlchen/chain/ethereum/manager.py

```diff
@@ -974,3 +974,34 @@ def get_blocknumber_by_time(self, ts: Timestamp, etherscan: bool = True) -> int:
             except RemoteError:
                 pass
         return self._get_blocknumber_by_time_from_subgraph(ts)
+
+    def get_basic_contract_info(self, address: ChecksumEthAddress) -> Dict[str, Any]:
+        """
+        Query a contract address and return basic information as:
+        - Decimals
+        - name
+        - symbol
+        if it is provided in the contract. This method may raise:
+        - BadFunctionCallOutput: If there is an error calling a bad address
+        """
+        # The only information we assume is decimals to be 18
+        info: Dict[str, Any] = {'decimals': 18}
+        request = ('decimals', 'symbol', 'name')
+
+        def query_info(method: str) -> Union[str, int, None]:
+            try:
+                return self.call_contract(
+                    contract_address=address,
+                    abi=ERC20TOKEN_ABI,
+                    method_name=method,
+                )
+            except (BlockchainQueryError, RemoteError):
+                return None
+
+        for prop in request:
+            # In python3.8 we can use walrus operator
+            value = query_info(prop)
+            if value is not None:
+                info[prop] = value
```

Comment:
Hmm also I see that if something is missing you don't add the key in the response. Can you instead **always** add it but simply as `None`? This will be more explicit and easier to read in the frontend.